content/release/ReleaseNotes372.md

```diff
@@ -0,0 +1,107 @@
+<!--
+title: "Contrast 3.7.2 - March 2020"
+description: "Contrast 3.7.2 March 2020"
+tags: "3.7.2 March Monthly Release News"
+-->
+
+## On-premises release date:
+March 31, 2020
+
+### New features and improvements
+
+**Python for Assess**
+This feature is now generally available for all Assess users to instrument web applications written in Python to find custom code vulnerabilities. Assess capabilities include support for Django, Flask, and Pyramid along with route coverage and route-based auto-verification for each framework.
+
+### Important note
+With the 3.7.1 release, which upgraded Tomcat to version 9, users were experiencing a broken AJP (Apache JServ Protocol) connection. This required an AJP secret, or that the secret be disabled. This is configurable through server.properties.
```

Comment:
I think the last sentence could be: This is configurable through the _server.properties_ file.

Do you want me to edit the .md to make this change?

Reply:
This has been completed.